package.json

```diff
@@ -134,6 +134,7 @@
     "webpack-dev-server": "^3.11.2"
   },
   "peerDependencies": {
+    "graphql": "^15.0.0 || ^16.0.0",
```

Comment:
But one concern I do have is to double-check MSW's tree shaking. 

1. List this as an optional peer dependency.
2. Create an empty project and install MSW there.
3. Don't use `graphql`. Would your app compile?

Reply:
This is what you've confirmed with the sandbox, right, @JohnDaly?

Reply:
Yes, I was able to run the Sandbox and use MSW to mock REST requests